analytics.go

```diff
@@ -283,13 +302,13 @@ func (c *Client) loop() {
 			msgs = append(msgs, msg)
 			if len(msgs) == c.Size {
 				c.verbose("exceeded %d messages – flushing", c.Size)
-				c.send(msgs)
+				go c.send(msgs)
```

Comment:
Hmm yeah might not be a bad idea, though hopefully we're never down enough for it to back up like mad with retries haha. Maybe just a second goro for handling retries with a high watermark that just drops after, not sure